ntbs-service/Pages/Notifications/NotificationsBase.cshtml.cs

```diff
@@ -65,14 +59,6 @@ public async Task<IActionResult> OnPostSubmitAsync()
             return RedirectToOverview();
         }
 
-        public async Task<IActionResult> OnPostCreateLinkAsync()
```

Comment:
This is now on OverviewModel as that's the only place this can actually be called from